R/utils.R

```diff
@@ -101,12 +102,11 @@ is_logical_vector_modif <- function(x, min_length = 1) {
 #'
 #' @export
 rtables_aligns <- function() {
-  c("left", "right", "center")
+  list_valid_aligns()
```

Comment:
deprecate this in favor of `formatters::list_valid_aligns`

Reply:
We should find out of this breaks anything downstream, if so, if so it should be deprecated right after the august release, if not we should deprecate before the release.

Reply:
I checked. It is never used downstream as far as I could find with staged.deps and manual search